Reviews/337 -  Royal web wallet and casper connector/1- Development/README.md

```diff
@@ -0,0 +1,394 @@
+
```

Comment:
Please fix the path. It should be "1 - Development" instead of "1- Development".

Reply:
done